response/slack/decorators/headline_post_action.py

```diff
@@ -29,9 +29,6 @@ def _wrapper(fn):
         SLACK_HEADLINE_POST_ACTION_MAPPINGS[
             order
         ] = SLACK_HEADLINE_POST_ACTION_MAPPINGS.get(order, []) + [fn]
-        logger.info(
-            f"Registering headline post action {fn.__name__} with order {order}"
-        )
```

Comment:
This removal is unrelated (sorry), but it spams every time the server starts up so not the most useful to have in the logs.